js/wp-autoupdates.js

```diff
@@ -2,100 +2,114 @@
 ( function( $, settings, pagenow ) {
 	'use strict';
 
-	$( document ).ready( function() {
-		$( '.autoupdates_column, .theme-overlay' ).on( 'click', 'a.auto-update', function( event ) {
-			var data,
-				$anchor = $( this ),
-				type    = $anchor.attr( 'data-wp-type' ),
-				action  = $anchor.attr( 'data-wp-action' ),
-				$label  = $anchor.find( '.label' ),
-				$parent = $anchor.parents( 'themes' !== pagenow ? '.autoupdates_column' : '.theme-autoupdate' );
+	$( document ).ready(
+		function() {
+			$( '.autoupdates_column, .theme-overlay' ).on(
+				'click',
+				'a.auto-update',
+				function( event ) {
+					var data,
+					$anchor = $( this ),
+					type    = $anchor.attr( 'data-wp-type' ),
+					action  = $anchor.attr( 'data-wp-action' ),
+					$label  = $anchor.find( '.label' ),
+					$parent = $anchor.parents( 'themes' !== pagenow ? '.autoupdates_column' : '.theme-autoupdate' );
 
-			event.preventDefault();
+					event.preventDefault();
 
-			// Clear any previous errors.
-			$parent.find( '.auto-updates-error' ).removeClass( 'notice error' ).addClass( 'hidden' );
+					// Clear any previous errors.
+					$parent.find( '.auto-updates-error' ).removeClass( 'notice error' ).addClass( 'hidden' );
 
-			// Show loading status.
-			$label.text( 'enable' === action ? wp_autoupdates.enabling : wp_autoupdates.disabling );
-			$anchor.find( '.dashicons-update' ).removeClass( 'hidden' );
+					// Show loading status.
+					$label.text( 'enable' === action ? wp_autoupdates.enabling : wp_autoupdates.disabling );
+					$anchor.find( '.dashicons-update' ).removeClass( 'hidden' );
 
-			data = {
-				action: 'toggle_auto_updates',
-				_ajax_nonce: settings.ajax_nonce,
-				state: action,
-				type: type,
-				asset: $anchor.attr( 'data-wp-asset' ),
-			};
+					data = {
+						action: 'toggle_auto_updates',
+						_ajax_nonce: settings.ajax_nonce,
+						state: action,
+						type: type,
+						asset: $anchor.attr( 'data-wp-asset' ),
+					};
 
-			$.post( ajaxurl, data )
-				.done( function( response ) {
-					var $enabled, $disabled, enabledNumber, disabledNumber;
+					$.post( ajaxurl, data )
+					.done(
```

Comment:
are these auto-fixes produced by tooling or did you do those by hand?  I ask because the Ajax chaining stuff doesn't seem to conform to https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#chained-method-calls.  The indentation doesn't look right to me.

And is it really necessary to have the anonymous functions on their own line?

Reply:
`phpcbf --standard=WordPress-Core --report-summary,source .`

This is similar to the code sniffs that are run on commit.

Reply:
OK, fine by me then (altho it makes the JS harder to read, IMHO :-)